contents/docs/api/overview.mdx

```diff
@@ -64,9 +64,9 @@ https://posthog.example.com/api/person/
 
 <br />
 
-## `/user/`
+## `/users/@me`
 
-The [`/user/` endpoint](./user) gives you a lot of useful information about the possible event names and properties you can use throughout the rest of the API. 
+The [`/users/@me` endpoint](./user) gives you a lot of useful information about the possible event names and properties you can use throughout the rest of the API.
```

Comment:
Well this statement no longer holds true actually, all the event and properties information is now sent in the `/api/event_definition/` and `/api/property_definition` endpoints. Though this endpoint does contain a lot of other useful information for the current user.

Reply:
Thanks, I didn't know where they are, will update